config/routes.rb

```diff
@@ -0,0 +1,28 @@
+Rails.application.routes.draw do
+
+  post "articles/:id/update" => "articles#update"
+  get "articles/:id/edit" => "articles#edit"
+  post "articles/create" => "articles#create"
```

Comment:
これでもOKですが、`resources`を使った書き方のほうがRailsらしくてきれいだと思います。
```suggestion
  resources :articles, except: :index
```

Reply:
お恥ずかしながらresourcesを使うことで
こんなに簡略化できることを知りませんでした。ありがとうございます。
修正しました。